plaid/src/bin/config_check.rs

```diff
@@ -0,0 +1,27 @@
+use clap::{Arg, Command};
+use plaid::config::read_and_interpolate;
+
+fn main() {
+    env_logger::init();
+    let matches = Command::new("Config Check")
```

Comment:
I tried running help for this but couldn't get it to function. Not sure if there is something wrong with the invocation but would be nice to get help to show up by default.